onnxscript/type_annotation.py

```diff
@@ -43,13 +43,31 @@
 
 _LIST_CONSTRUCTORS = frozenset([list, typing.List, typing.Sequence, collections.abc.Sequence])
 
+# A sorted list of all type strings used in an OpSchema
+ALL_TENSOR_TYPE_STRINGS = (
```

Comment:
nit: must this be hardcoded?

Reply:
Not sure where else we can find it?

Reply:
I guess you could compute it by iterating over [tensor_type_registry](https://github.com/microsoft/onnxscript/blob/eb58e24b81b84c1a1d54a66deca6c8ebba8576e4/onnxscript/onnx_types.py#L35) ... it would be convenient to have the `to_string` method defined in one of the other PRs.

Reply:
But do you want the 8-bit floats included in this?

Reply:
Done